wallets/metamask/src/fixtures/metaMaskFixtures.ts

```diff
@@ -0,0 +1,155 @@
+import { MetaMask, getExtensionId, prepareExtension, unlockForFixture } from '../../src'
+
+import path from 'node:path'
+import { type Page, chromium } from '@playwright/test'
+import { test as base } from '@playwright/test'
+import {
+  CACHE_DIR_NAME,
+  createTempContextDir,
+  defineWalletSetup,
+  removeTempContextDir
+} from '@synthetixio/synpress-core'
+import { type Anvil, type CreateAnvilOptions, createPool } from '@viem/anvil'
+import fs from 'fs-extra'
```

Comment:
why do we need `fs-extra` here? could we use `node:fs/promises`?

Reply:
i have also seen reference to `fs` somewhere, we can use `node:fs` instead

Reply:
We use this package in the monorepo already so it should not change much if we use it here as well. We can consider to remove it completely - that would make sense.